deep_speech_2/infer.py

```diff
@@ -103,8 +104,7 @@ def infer():
 
     error_rate_func = cer if args.error_rate_type == 'cer' else wer
     target_transcripts = [
-        ''.join([data_generator.vocab_list[token] for token in transcript])
-        for _, transcript in infer_data
+        transcript for _, transcript in infer_data
```

Comment:
Line 106 - 108 could be put into one line ?

Reply:
Done.